src/ape/api/networks.py

```diff
@@ -2,11 +2,17 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Optional, Type
 
+from eth_account import Account as EthAccount  # type: ignore
+from eth_account._utils.legacy_transactions import (
```

Comment:
Since we are using `==` in setup.py for the eth-account version, this is not a big deal. I think we have a ticket in the backlog for replacing this elsewhere too anyways.